data/datastore/src/main/proto/com/eblan/launcher/data/datastore/proto/appdrawer/eblan_application_info_order.proto

```
@@ -0,0 +1,9 @@
+syntax = "proto3";
```

Buf (1.65.0)

[error] Files must have a package defined.

(PACKAGE_DEFINED)